slides.md

```diff
@@ -574,82 +580,71 @@ All files |        0 |        0 |        0 |        0 |                   |
 
 ---
 
-# Step 6: Validation
+# Step 6: Serialization
 
-- Route validation internally relies upon [Ajv](https://www.npmjs.com/package/ajv), which is a high-performance JSON Schema validator
+- Fastify uses a schema-based approach, and even if it is not mandatory we recommend using JSON Schema to validate your routes and serialize your outputs. Internally, Fastify compiles the schema into a highly performant function
 
-https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/#validation
+- We encourage you to use an output schema, as it can drastically increase throughput and help prevent accidental disclosure of sensitive information
 
+https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/
 ---
 
 # Step 6: Exercise 💻
 
-- Create and register a `POST /login` route in `routes/login.js`
+- Validate the response in the users route using a schema:
 
-- Validate the body of the request to ensure it is a JSON object containing two required string properties: `username` and `password`
+  - Created with [`fluent-json-schema`](https://github.com/fastify/fluent-json-schema)
```

Comment:
This explanation should be moved to the validation step.

Reply:
@mcollina Here The JSON Schema is used to validate the response, before this pr, the validation of the response was done in the serialization step. Should I merge with the validation step? In this case, moving the response validation, our serialization step will be empty.

Reply:
What I'm referring to is fluent-json-schema. This should be introduced earlier in the slides as it is used before.